spec_synapse_v2.md

```diff
@@ -413,9 +413,21 @@ dataset size: 3xS
 datatype: DOUBLE
 
 
+### nrn_line
+description: Synapse position in corresponding postsynaptic GID dataset in nrn.h5.
+Used for retro-compatibility with NRN; would be dropped eventually.
+
+This property is optional
+
+
+dataset size: 1xS
+
+datatype: INT32
```

Comment:
Little comments on that:
For now every other dataset uses INT64 or UINT64 for integer. 
Could you use UINT64 for this one like it is a line number ?
It would avoid to have to support one more datatype just for nrn_line.

Space does not really matter anyway, Like it is line numbers anyway, it should compress beautifully if needed.